build_tag.sh

```diff
@@ -2,9 +2,11 @@
 
 set -exvo pipefail -o nounset
 
-docker login quay.io -u ${QUAY_USER} -p ${QUAY_TOKEN}
+# utilize local go 1.17 version if available
+GO_1_17="/opt/go/1.17.5/bin"
 
-IMAGE=ocm-addons-ci
+if [ -d  "${GO_1_17}" ]; then
```

Comment:
what would be the default option?

Reply:
If the CI worker does not have the specified version the `PATH` will remain unchanged and use the default `go` installation. This also retains compatibility with local development where the first `go` command encountered on the `PATH` should be used.